openpype/modules/ftrack/ftrack_module.py

```diff
@@ -479,6 +482,50 @@ def cli(self, click_group):
         click_group.add_command(cli_main)
 
 
+def _check_ftrack_url(url):
+    import requests
+
+    try:
+        result = requests.get(url, allow_redirects=False)
+    except requests.exceptions.RequestException:
+        return False
+
+    if (result.status_code != 200 or "FTRACK_VERSION" not in result.headers):
+        return False
+    return True
+
+
+def check_ftrack_url(url, log_errors=True, logger=None):
+    """Checks if Ftrack server is responding"""
+
+    if logger is None:
+        logger = Logger.get_logger(__name__)
+
+    url = url.strip("/ ")
+    if not url:
+        logger.error("Ftrack URL is not set!")
+        return None
+
+    if not url.startswith("http"):
+        url = "https://" + url
+
+    ftrack_url = None
+    if not url.endswith("ftrackapp.com"):
+        ftrackapp_url = url + ".ftrackapp.com"
+        if _check_ftrack_url(ftrackapp_url):
+            ftrack_url = ftrackapp_url
+
+    if not ftrack_url and _check_ftrack_url(url):
+        ftrack_url = url
+
+    if ftrack_url:
+        logger.debug("Ftrack server \"{}\" is accessible.".format(ftrack_url))
+    elif log_errors:
+        logger.error("Entered Ftrack URL \"{}\" is not accesible!".format(url))
```

Comment:
```suggestion
        logger.error("Entered Ftrack URL \"{}\" is not accessible!".format(url))
```

Or even better and align more with the other message?
```suggestion
        logger.error("Ftrack server \"{}\" is not accessible!".format(url))
```